CONTRIBUTING.md

```diff
@@ -90,9 +90,9 @@ Gradle will throw an exception during configuration time: "task clean cannot be
     The plugin should be referred via the "plugin id", examples:
 
 ```Groovy
-apply plugin: "org.shipkit.continuous-delivery"
+apply plugin: "org.shipkit.java"
```

Comment:
Absolutely! Let's keep the String identifiers of plugins in sync with the implementing class name. I suggest to call the new class "ShipkitJavaPlugin".